cdk/lib/Airdrop/cdk-pipeline-stack.ts

```diff
@@ -63,6 +60,6 @@ export class CDKPipelineStack extends Stack {
     super(scope, id, props);
 
     createPipeline(this, appEnv.ropsten);
```

Comment:
The code is going to mainnet, do we need to refer to mainnet instead of ropsten.